view-creator-framework/src/main/java/org/hypertrace/core/viewcreator/pinot/PinotViewCreatorConfig.java

```diff
@@ -0,0 +1,38 @@
+package org.hypertrace.core.viewcreator.pinot;
+
+public class PinotViewCreatorConfig {
```

Comment:
```suggestion
public class PinotViewCreatorConfigConstants {
```

Reply:
We are using Config as Suffix for such constants (JobConfig)